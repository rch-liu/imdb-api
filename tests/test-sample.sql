SELECT films.original_title, ratings.average_ratings 
FROM ratings, films
WHERE films.start_year > 1894 AND
      films.start_year < 1897 AND
      ratings.tconst = films.tconst AND
      ratings.num_votes >= 10
ORDER BY ratings.average_ratings DESC
LIMIT 10