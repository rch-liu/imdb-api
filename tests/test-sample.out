           original_title            | average_ratings 
-------------------------------------+-----------------
 Dancing Darkies                     |             7.8
 L'arrivée d'un train à La Ciotat    |             7.4
 L'arroseur arrosé                   |             7.1
 La sortie de l'usine Lumière à Lyon |             6.9
 A Watermelon Feast                  |             6.9
 Un petit diable                     |             6.8
 La voiture du potier                |             6.7
 Le manoir du diable                 |             6.7
 Danse serpentine                    |             6.5
 Démolition d'un mur                 |             6.4
(10 rows)